src/uct/sm/mm/base/mm_iface.c

```diff
@@ -29,6 +29,10 @@ static ucs_config_field_t uct_mm_iface_config_table[] = {
      "Size of the receive FIFO in the memory-map UCTs.",
      ucs_offsetof(uct_mm_iface_config_t, fifo_size), UCS_CONFIG_TYPE_UINT},
 
+    {"SEG_SIZE", "8k",
+     "Maximal size of the receive descriptor (for payload).",
```

Comment:
"Size of send/receive buffers for copy-out sends"

Reply:
fixed